homeassistant/components/tuya/switch.py

```diff
@@ -573,6 +573,13 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Air Conditioner Mate (Smart IR Socket)
+    "wnykq": (
+        SwitchEntityDescription(
+            key=DPCode.SWITCH,
+            name="Switch",
```

Comment:
Should it be named "Switch" in this case? Or just after the device name?

Reply:
IMHO the standard "switch" is ok in this case. I changed to `translation_key="switch"`.

Reply:
But it is the only switch in the device right?

Reply:
Right. This device is an Air Conditioner companion. It is a socket with a power monitor and IR receiver/transmitter. The device does not contain a relay. This `single` virtual switch is used to enable the Air Conditioner control.

Reply:
ok, clear, in that case we should remove the name and let it adopt the device name.